README.md

```diff
@@ -258,6 +258,7 @@ examples = null
 | Argument                  | Type                                    | Default      | Description                                                                                                                                  |
 | ------------------------- | --------------------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
 | `labels`                  | `str`                                   |              | Comma-separated list of labels.                                                                                                              |
+| `template`                | `Union[str, Callable[[], str]]`         |              | String or a function that returns the prompt template to render.                                                                             |
```

Comment:
Unsure of this. There is a default, actually, the `_DEFAULT_NER_TEMPLATE`. Not sure what to put in the `Default` column.

Reply:
Hm, we could move _DEFAULT_NER_TEMPLATE into a registry function and specify the registry handle of that function here? Although it seems odd to do this just for the docs. Alternatively a "See here" linking to `_DEFAULT_NER_TEMPLATE` in the code?

Reply:
I like the latter suggestion!